bin/npm.cmd

```diff
@@ -1,17 +1,6 @@
 @echo off
 setlocal EnableDelayedExpansion
 
-:: Try to find git.exe in path
```

Comment:
Why is this removed?  It was added so that people on Windows using APM on the command line installing packages that look for git.exe could actually find the one installed by Git Desktop because it isn't in the path...

It shouldn't be interfering with AppVeyor builds in any way.